core/modules/commands/savewikifolder.js

```diff
@@ -5,7 +5,7 @@ module-type: command
 
 Command to save the current wiki as a wiki folder
 
---savewikifolder <wikifolderpath> [<filter>]
+--savewikifolder <wikifolderpath> [<name>=<value>]
```

Comment:
It breaks backwards compatibility to remove the existing optional `<filter>` parameter. I think the new syntax should be:

```
--savewikifolder <wikifolderpath> [<filter>] [ [<name>=<value>] ]*
```

Reply:
Yes, this breaks backward compatibility, but it also adds complexity because the filter is also an optional parameter. Can we put the filter inside `[<name>=<value>]`?

Reply:
<wikifolderpath> [<filter>] [ [<name>=<value>] ]* There doesn't seem to be a way to do this, what I can think of is that unless the filter is made mandatory so that it is accurate and simple to get, we may need to test one by one if the parameter is a filter parameter.

Reply:
Backward compatibility support I solved

Reply:
Well, it's been solved